dev-build/conf/routes

```diff
@@ -10,6 +10,9 @@ GET            /reader-revenue/contributions-banner
 POST           /reader-revenue/redeploy-contributions-banner/$region<united-kingdom|united-states|australia|rest-of-world>                                               controllers.admin.ReaderRevenueAdminController.redeployContributionsBanner(region)
 GET            /reader-revenue/contributions-banner-deploy-log/$region<united-kingdom|united-states|australia|rest-of-world>                                             controllers.ReaderRevenueAppController.contributionsBannerDeployLog(region: String)
 
+# FONT LOADER
+GET            /font-loader                                                                                                      controllers.FontLoaderController.renderFontLoader()
+
```

Comment:
Is it needed in `dev-build`?

Reply:
No I don't think so, I can remove this now

Reply:
Whether this stays in `dev-build` or not, it's probably worth documenting how to run locally with Support.

Reply:
I think by default, Support should get its fonts from the PROD version of frontend (unless you specifically override it). Otherwise wouldn't this mean you'd have to run Frontend every time you run Support?

Reply:
I guess that's down to Support, but if they want to run their site locally without having to run `frontend` locally then they'll need to do as @SiAdcock suggested and point to PROD. I'm going to include some docs with this PR and how to integrate with the PROD endpoint.

Reply:
Yes I had presumed we would use the PROD endpoint